packages/sdk-ts/src/utils/constants.ts

```diff
@@ -47,4 +47,36 @@ export const CW20_SWAP_CONTRACT_BY_NETWORK = {
   [Network.Local]: 'inj177yh38g3ctu7cemxpa3c2kvwh2yslfxfmfa66h',
 }
 
+export const CW20_STAKING_CONTRACT_BY_NETWORK = {
+  [Network.Mainnet]: '',
+  [Network.MainnetLB]: '',
+  [Network.MainnetK8s]: '',
+  [Network.Public]: '',
+  [Network.Staging]: '',
+  [Network.Testnet]: 'inj1pxzykc8qry3ytxwxr3ua72tn6e4wvusj40yy2w',
+  [Network.TestnetK8s]: 'inj1pxzykc8qry3ytxwxr3ua72tn6e4wvusj40yy2w',
+  [Network.TestnetOld]: 'inj1pxzykc8qry3ytxwxr3ua72tn6e4wvusj40yy2w',
+  [Network.Devnet]: 'inj15r735cjk9m6ujxtqn9zgxtdhd2lfzslde6c0s5',
+  [Network.Devnet1]: 'inj15r735cjk9m6ujxtqn9zgxtdhd2lfzslde6c0s5',
+  [Network.Devnet2]: 'inj15r735cjk9m6ujxtqn9zgxtdhd2lfzslde6c0s5',
+  [Network.Local]: 'inj15r735cjk9m6ujxtqn9zgxtdhd2lfzslde6c0s5',
+}
+
+export const CW20_ALLOCATOR_CONTRACT_BY_NETWORK = {
+  [Network.Mainnet]: '',
+  [Network.MainnetLB]: '',
+  [Network.MainnetK8s]: '',
+  [Network.Public]: '',
+  [Network.Staging]: '',
+  [Network.Testnet]: 'inj1qsft9ptmywulv5hjzufl77j784sr9hf5yj6fwx',
+  [Network.TestnetK8s]: 'inj1qsft9ptmywulv5hjzufl77j784sr9hf5yj6fwx',
+  [Network.TestnetOld]: 'inj1qsft9ptmywulv5hjzufl77j784sr9hf5yj6fwx',
+  [Network.Devnet]: 'inj1nu9wf9dw384attnpu0pwfet5fajn05w2xp99u3',
+  [Network.Devnet1]: 'inj1nu9wf9dw384attnpu0pwfet5fajn05w2xp99u3',
+  [Network.Devnet2]: 'inj1nu9wf9dw384attnpu0pwfet5fajn05w2xp99u3',
+  [Network.Local]: 'inj1nu9wf9dw384attnpu0pwfet5fajn05w2xp99u3',
+}
+
+export const CW20_MITO_MASTER_CONTRACT_CODE_ID = 1573
```

Comment:
Let's remove `CW20_MITO_MASTER_CONTRACT_CODE_ID`, feels like an overkill housing this in sdk-ts

Reply:
@ThomasRalee, removed the `CW20_MITO_MASTER_CONTRACT_CODE_ID` from `constants.ts`